src/net/tcp.rs

```diff
@@ -208,12 +209,6 @@ impl TcpListener {
     }
 }
 
-impl fmt::Debug for TcpListener {
```

Comment:
Could you explain why you changed this implementation? The existing implementation is a bit cleaner.

Reply:
Well now the contents of the struct is not really being displayed, which can be deceiving when debugging problems. Just using derive is just as simple, while actually display the contents. Is there a reason this was written by hand?

Reply:
IMO the following isn't super useful:

```rust
TcpStream { inner: TcpStream { inner: TcpStream } }
```

I believe that it was written by hand to avoid this sort of output.